internal/handlers/handlers.go

```diff
@@ -1 +1,59 @@
 package handlers
+
+import (
+	"github.com/Yandex-Practicum/go1fl-sprint6-final/internal/service"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+)
+
+func IndexHandler(w http.ResponseWriter, r *http.Request) {
+	data, err := os.ReadFile("index.html")
+	if err != nil {
+		http.Error(w, "Не удалось загрузить HTML", http.StatusInternalServerError)
```

Comment:
Это проблема клиента
```suggestion
		http.Error(w, "Не удалось загрузить HTML", http.StatusBadRequest)
```